### DISCUSSION TITLE
84026815-5393-46cf-994c-d8b3d2fb1ce0

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#10] [General]
# 84026815-5393-46cf-994c-d8b3d2fb1ce0

우테코 미션을 하다보면, 추상클래스를 쓸 때도 있고 인터페이스를 쓸 때도 있다. 둘의 차이를 내 생각과 함께 면밀히 정리해보겠다. 용어 정의에 바탕한 이론을 다루는 글임을 미리 밝힌다. 학창 시절 교수님께 배웠던 C++의 추상클래스와 비교하며 원론적인 정의를 내려보겠다.

https://choo.oopy.io/84026815-5393-46cf-994c-d8b3d2fb1ce0

**Comment by jongmee** [comment]
저는 C++을 깊게는 몰랐지만, 잘 이해될 만큼 좋은 글이었어요 👍 배워 갑니다!
